lib/android-helpers.js

```diff
@@ -467,11 +467,11 @@ helpers.initDevice = async function (adb, opts) {
   }
   if (!opts.avd) {
```

Comment:
then it should be another bug. We have to synchronize **avd** capability value in such case